modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2507,6 +2508,17 @@
 		end
 	end
 
+	function m.sanitizers(cfg)
+		if _ACTION >= "vs2022" and cfg.sanitize then
+			if table.contains(cfg.sanitize, "Address") then
```

Comment:
Is ASAN 2022+ only as well? I thought it was VS 2019+

Reply:
Specifically VS2019 16.9, because it isn't supported in earlier VS versions and I have no way to check the specific version I rounded up to the next thing we can actually check.